test/runtime_tests.cc

```diff
@@ -763,6 +763,39 @@ namespace detail {
 		REQUIRE_FALSE(bm.has_active_buffers());
 	}
 
+	TEST_CASE("buffer_manager allows buffers deallocation", "[buffer_manager][dealloc]") {
+		celerity::distr_queue q;
+		buffer_id b_id;
+		auto& bm = runtime::get_instance().get_buffer_manager();
+		auto& tm = runtime::get_instance().get_task_manager();
+		int new_horizon_step = 2;
```

Comment:
This is never modified and should probably be `const`, or even `constexpr`.

Reply:
Or it could just be a literal that is passed into `set_horizon_step` directly...

Reply:
@psalz it's used for computing the number of tasks to submit in the loop below - which I would like to keep.

Reply:
You're right, sorry!